API/Import.php

```diff
@@ -133,16 +134,20 @@ public function importContainerVersion($exportedContainerVersion, $idSite, $idCo
         $ecv = $exportedContainerVersion;
 
         foreach ($ecv['variables'] as $variable) {
-            Request::processRequest('TagManager.addContainerVariable', array(
-                'idSite' => $idSite,
-                'idContainer' => $idContainer,
-                'idContainerVersion' => $idContainerVersion,
-                'type' => $variable['type'],
-                'name' => $variable['name'],
-                'parameters' => $variable['parameters'],
-                'defaultValue' => $variable['default_value'],
-                'lookupTable' => $variable['lookup_table'],
-            ));
+            try {
+                Request::processRequest('TagManager.addContainerVariable', array(
+                    'idSite' => $idSite,
+                    'idContainer' => $idContainer,
+                    'idContainerVersion' => $idContainerVersion,
+                    'type' => $variable['type'],
+                    'name' => $variable['name'],
+                    'parameters' => $variable['parameters'],
+                    'defaultValue' => $variable['default_value'],
+                    'lookupTable' => $variable['lookup_table'],
+                ));
+            } catch (EntityRecursionException $e){
+                throw new \Exception(Piwik::translate('TagManager_EntityRecursionExceptionForVariable', array($variable['name'] . '(' . $variable['type'] . ')')));
```

Comment:
@tsteur Should we capture all such exceptions in a Array lets say and based on that throw an exception or 1 at a time is fine ?